Library/Homebrew/cleanup.rb

```diff
@@ -214,7 +214,6 @@ def cleanup_formula(formula, quiet: false)
       formula.eligible_kegs_for_cleanup(quiet: quiet)
              .each(&method(:cleanup_keg))
       cleanup_cache(Pathname.glob(cache/"#{formula.name}--*"))
-      rm_ds_store([formula.rack])
```

Comment:
I think ideally a new argument e.g. `def cleanup_formula(formula, quiet: false, rm_ds_store: true)` (and same for `cleanup_cask`) would be added and then `rm_ds_store([formula.rack]) if rm_ds_store`. That means `clean!` above could call `cleanup_formula(formula, quiet: quiet, rm_ds_store: false)` to avoid the double removal but it would still remove DS_Store on a single formula/cask cleanup. Thoughts?